File: back/src/main/java/com/back/domain/concert/repository/ConcertDeatilRepository.java
package com.back.domain.concert.repository;

import com.back.domain.concert.entity.ConcertDetail;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ConcertDeatilRepository extends JpaRepository<ConcertDetail, Long> {

    List<ConcertDetail> findByConcertConcertId(Long concertId);
}

File: back/src/main/kotlin/com/back/domain/concert/repository/ConcertDeatilRepository.kt
package com.back.domain.concert.repository

import com.back.domain.concert.entity.ConcertDetail
import org.springframework.data.jpa.repository.JpaRepository

interface ConcertDeatilRepository : JpaRepository<ConcertDetail, Long> {
    fun findByConcertConcertId(concertId: Long): List<ConcertDetail>
}
